ivy/functional/frontends/numpy/creation_routines/numerical_ranges.py

```diff
@@ -156,3 +156,27 @@ def __init__(self):
 
 
 ogrid = OGrid()
+
+
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+def geomspace(start, stop, num=50, endpoint=True, dtype=None, axis=0):
+    # Check if stop < start and swap values if necessary
+    flip = 0
+    if stop < start:
+        start, stop = stop, start
+        flip = 1
```

Comment:
This won't work with multi-dimensional inputs, start and stop don't have to be scalars, they can also be arrays, you can probably perform something similar by using `where` and the axis argument

Reply:
Can I instead use "for i in range(start)" and keep the entire code as is
and then at the end just append() each row